code/modules/assembly/flash.dm

```diff
@@ -243,7 +243,8 @@
 	if(generic_message && M != user)
 		to_chat(M, "<span class='disarm'>[src] emits a blinding light!</span>")
 	if(targeted)
-		if(M.flash_act(1, 1))
+		//No flash protection, blind and stun
```

Comment:
Knockdown?

Reply:
The flash will apply knockdown to targets with low stamina damage, targets with high stamina damage will be stunned